docs/sources/developers/plugins/create-a-grafana-plugin/develop-a-plugin/sign-a-plugin.md

```diff
@@ -30,7 +41,7 @@ To verify ownership of your plugin, generate an API key that you'll use every ti
 
 Public plugins need to be reviewed by the Grafana team before you can sign them.
 
-1. Submit your plugin for [review]({{< relref "publish-a-plugin/#publishing-your-plugin-for-the-first-time" >}})
+1. Submit your plugin for [review]({{< relref "../../publish-a-plugin/publish-or-update-a-plugin.md#publish-your-plugin" >}})
```

Comment:
The very fact that in order to sign as community/commercial you _have_ to go to the publishing docs to me also adds to the argument to keep them together